### ISSUE TITLE
Maintainers Wanted

### ISSUE THREAD

**Issue by exAspArk** [uniiverse/apollo-tracing-ruby#17]
We are looking for people to help out and maintain the gem since our company no longer uses [Apollo Engine](https://www.apollographql.com/engine/).

Please comment on this issue if you're interested.

**Comment by jturkel** [comment]
I'm still evaluating Apollo Engine but I'd be happy to help out if we decide to move forward with it.

**Comment by bezoar17** [comment]
@exAspArk just curious, what do you guys use now instead of Apollo Engine for insights ?

**Comment by exAspArk** [comment]
@bezoar17 in our particular case, we switched to Skylight since we use it for non-GraphQL instrumentations. Luckily, GraphQL-Ruby gem supports the tools (AppSignal, New Relic, Scout, Skylight, Datadog, Prometheus) out of the box with just 1 line of code http://graphql-ruby.org/queries/tracing.html.

**Comment by fgrehm** [comment]
@exAspArk thanks for the update! I personally came across this project after getting to know about the [Apollo Platform](https://www.apollographql.com/docs/intro/platform.html) and doing some digging for Ruby tooling. We want to leverage the [schema validation features](https://www.apollographql.com/docs/platform/schema-validation.html) they provide and even though I wasn't able to find any information whether they use the same tracing information provided by this gem, we have interest in pursuing that if time permits (and maybe use this gem as a baseline if we can)

If anyone else has interest in that type of thing please reach out to me, I might not have the time to implement but would be more than happy to be a beta tester 🍻

**Comment by glasser** [comment]
Hi @fgrehm! The schema validation features of the Apollo Platform work with any GraphQL server with no instrumentation required. The features are improved if you're instrumenting your server and sending metrics (since you can tell if changed fields are being actively used by clients) but you can get started with any GraphQL server.

**Comment by fgrehm** [comment]
> The features are improved if you're instrumenting your server and sending metrics (**_since you can tell if changed fields are being actively used by clients_**)

Yeah, that part I highlighted in bold is exactly what we're interested on! I thought we'd need something like this project to collect those metrics 🤔

**Comment by modosc** [comment]
definitely interested in helping out. we're not using `apollo-engine` but we are using this gem with `graphql-playground`

**Comment by lsanwick** [comment]
Just wanted to put the message out to interested folks that I have released https://github.com/EnjoyTech/apollo-studio-tracing-ruby, which is fully compatible with the new Apollo Studio tracing format.

Big shout out to @jturkel for doing a lot of the heavy lifting on this branch: https://github.com/salsify/apollo-tracing-ruby/tree/feature/new-apollo-api
